packages/data-row-permissions/vue.config.js

```diff
@@ -0,0 +1,75 @@
+const PROXY_TARGET = 'https://preview-pr-9315-12.dev.molgenis.org/' // 'https://master.dev.molgenis.org'
```

Comment:
bit chicken and egg what the right order is, but I'd already check in master.dev here because otherwise we need another PR after merging

Reply:
good catch, although not breaking :) I'll change it back!